sdk/cognitiveservices/cognitiveservices-qnamaker/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/cognitiveservices-qnamaker",
   "author": "Microsoft Corporation",
   "description": "QnAMakerClient Library with typescript type definitions for node.js and browser.",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
It doesnt look like there are any breaking changes to the updated SDK. Is a major version update needed? I see new apis being added, perhaps a minor version update is enough?

Reply:
Makes sense, changed the version to 3.1.0